### ISSUE TITLE
v2 solar charger is missing main connectors

### ISSUE THREAD

**Issue by fgnievinski** [adafruit/Fritzing-Library#153]
LOAD and BATT cannot be wired.
only the side pins are wireable. 
![image](https://user-images.githubusercontent.com/5018500/140539045-db603786-6eb9-484b-8c4c-ef837acc93b1.png)

the newer (v3 USB-C) is fine.
![image](https://user-images.githubusercontent.com/5018500/140538760-b354dcb2-326b-44ca-9c70-4f583365c274.png)

**Comment by ladyada** [comment]
yeah we changed how we do fritzing files, we probably wont be going back to 'redo' old ones any time soon
